source/Audio.cpp

```diff
@@ -0,0 +1,103 @@
+#include <algorithm>
+#include <iostream>
+#include <thread>
+#include <chrono>
+#include "AudioListener.hpp"
+#include "AudioSource.hpp"
+#include "Audio.hpp"
+#include "Music.hpp"
+#include "Vect.hpp"
+#include "AudioError.hpp"
+
+std::array<char const *, static_cast<size_t>(Sounds::SIZE)> const Audio::soundFilenames
+{{
+  nullptr,
+  "resources/sounds/boyaux1.wav",
+  "resources/sounds/euuuh1.wav"
+}};
+std::array<char const *, static_cast<size_t>(Musics::SIZE)> const Audio::musicFilenames
+{{
+  nullptr,
+  "resources/musics/small_world.ogg"
+}};
```

Comment:
These two global variables seem nice candidates to be `constexpr`. I am aware this is a hassle however, so this is optional.